R/rater_functions.R

```diff
@@ -6,6 +6,11 @@
 ### randomize.order: soll die Reihenfolge der Pseudocodes nach Zufall bestimmt werden?
 make.pseudo <- function(datLong, idCol, varCol, codCol, valueCol, n.pseudo, randomize.order = TRUE, verbose = FALSE)   {
       datLong     <- eatTools::makeDataFrame(datLong)
+      lapply(c(idCol, varCol, codCol), checkmate::assert_scalar)
+      checkmate::assert_vector(valueCol)
```

Comment:
what kind of values does valueCol have? numeric or chr or both?

Reply:
this should be just another column identifier just as idcol, varcol etc. see man page